pkgs/development/compilers/gcc/common/configure-flags.nix

```diff
@@ -242,6 +242,10 @@ let
       #   cc1: error: fp software completion requires '-mtrap-precision=i' [-Werror]
       "--disable-werror"
     ]
+    ++ lib.optionals targetPlatform.isCygwin [
+      # doesn't cross-compile under Cygwin
```

Comment:
Do you mean "for Cygwin" (when running under some other platform)? Otherwise the check is wrong.